support-frontend/package.json

```diff
@@ -126,6 +126,7 @@
     "rimraf": "^2.6.2",
     "sass-loader": "^7.1.0",
     "sass-mq": "^5.0.0",
+    "uglifyjs-webpack-plugin": "^2.1.3",
```

Comment:
What is this needed for?

Reply:
this was added to allow vscode to read the import paths properly, I could add this to a different PR as that better for tracking